<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FolderSelection</class>
 <widget class="QWidget" name="FolderSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#FolderSelection {
    background-color: #FFFFFF;
    border-radius: 10px;
}

QLabel {
    font-size: 16px;
    font-weight: bold;
    color: #1E293B;
}

QLineEdit {
    border: 1px solid #E4E7EB;
    border-radius: 6px;
    padding: 10px;
    background-color: #F9FAFB;
    height: 20px;
    font-size: 13px;
    color: #64748B;
}

QPushButton {
    background-color: #3498DB;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 10px 15px;
    font-weight: bold;
    font-size: 13px;
}

QPushButton:hover {
    background-color: #2980B9;
}

QPushButton:pressed {
    background-color: #1F6AA5;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="folderLabel">
     <property name="text">
      <string>Thư mục chứa video</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="directoryLineEdit">
     <property name="placeholderText">
      <string>/Users/videos</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="browseButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Duyệt</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
